src/lib.rs

```diff
@@ -86,6 +86,8 @@ macro_rules! define_vector {
                 }
             }
 
+            /// Construct a vector from any type that can convert into the
```

Comment:
How about: `Construct this vector from any type which is convertible into the corresponding `mint` vector type.` or something of the sorts.